source/release/4.5.0/index.md

```diff
@@ -144,45 +151,62 @@ Fixing the host's OVN not being confiugred can be done by reinstalling it from e
    oVirt 4.5 has moved from abrt to systemd-coredump to gather crash dumps on the hypervisors
  - [BZ 2010205](https://bugzilla.redhat.com/show_bug.cgi?id=2010205) **vm_kill_paused_time value should be determined from io_timeout**
    Vdsm configuration option vars.vm_kill_paused_time was removed. The corresponding value is directly dependent on the value of recently introduced sanlock.io_timeout option and needn't be configured separately.
- - [BZ 1940824](https://bugzilla.redhat.com/show_bug.cgi?id=1940824) **[RFE] Upgrade OVN/OVS 2.11 in RHV to OVN/OVS 2.15**
+ - [BZ 1940824](https://bugzilla.redhat.com/show_bug.cgi?id=1940824) **[RFE] Upgrade OVN/OVS 2.11 in oVirt to OVN/OVS 2.15**
    Upgrade from OvS/OVN 2.11 to OVN 2021 and OvS 2.15.
 
 The upgrade is transparent to user as long as few conditions are met:
 
 1) The engine must be upgraded first.
```

Comment:
This list is rendering as a paragraph. You should mark each line with "-" or "1." so that it appears as a list.